app/src/types/generic/endpoints.ts

```diff
@@ -8,4 +8,6 @@ export enum endpoints {
     DOCUMENT_SEARCH = '/SearchDocumentReferences',
     DOCUMENT_UPLOAD = '/DocumentReference',
     DOCUMENT_PRESIGN = '/DocumentManifest',
+
+    LG_STITCH = '/LloydGeorgeStitch',
```

Comment:
You're going to love me for this one; there seems to be a mismatch in the naming convention between using Lloyd George and LG. I'm very much for not using abbreviated naming as lg will not mean much to new developers. 

What is the effort required to rename files / vars to lloyd geroge rather than lg at this stage?